src/tracer.cpp

```diff
@@ -123,6 +127,81 @@ double analyticsRate(TracerOptions options) {
   return std::nan("");
 }
 
+void startupLog(TracerOptions &options) {
+  auto env_setting = std::getenv("DD_TRACE_STARTUP_LOGS");
+  if (env_setting != nullptr && std::string(env_setting) == "0") {
+    // Startup logs are disabled.
+    return;
+  }
+  // C++17's filesystem api would be really nice to have right now..
+  std::string startup_log_path = "/var/tmp/dd-opentracing-cpp";
+  struct stat s;
+  if (stat(startup_log_path.c_str(), &s) == 0) {
```

Comment:
should we be logging these errors?

Reply:
It's being intentionally quiet about errors here, for the same reason the startup options aren't dumped to `stdout` / `stderr: it's likely to cause interference with apps using the tracer under relatively normal circumstances.